Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,24 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+function countWords(str) {
+  const normalizedString = str.toLowerCase().replace(/[.,!?]/g, "");
+  const words = normalizedString.split(/\s+/);
+
+  const wordCounts = {};
+  for (const word of words) {
+    if (word in wordCounts) {
```

Comment:
Curious why didn't you use the || 0 trick like in tally.js

Reply:
Both approaches work similarly.
 for (const word of words) {
    wordCounts[word] = (wordCounts[word] || 0) + 1;
  }